plugins/diff/windows/cimfs.go

```diff
@@ -77,11 +77,26 @@ func NewCimDiff(store content.Store) (CompareApplier, error) {
 // provided mounts. Archive content will be extracted and decompressed if
 // necessary.
 func (c cimDiff) Apply(ctx context.Context, desc ocispec.Descriptor, mounts []mount.Mount, opts ...diff.ApplyOpt) (d ocispec.Descriptor, err error) {
-	layer, parentLayerPaths, err := cimMountsToLayerAndParents(mounts)
+	if len(mounts) != 1 {
+		return ocispec.Descriptor{}, fmt.Errorf("%w: number of mounts should always be 1 for Windows layers", errdefs.ErrInvalidArgument)
```

Comment:
The idiom seems to be the wrapped error is at the end
```suggestion
		return ocispec.Descriptor{}, fmt.Errorf("number of mounts should always be 1 for Windows layers: %w", errdefs.ErrInvalidArgument)
```